src/flows/braze/update_flow.py

```diff
@@ -359,7 +369,9 @@ def mask_email_domain_outside_production(rows: List[Dict], email_column='EMAIL')
     return rows
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), result=get_s3_result()) as flow:
+with Flow(FLOW_NAME, schedule=schedule, executor=LocalDaskExecutor(), result=get_s3_result()) as flow:
+    dbt_run_result = dbt.DbtCloudRunJob()(cause=FLOW_NAME, job_id=DBT_CLOUD_BRAZE_JOB_ID, wait_for_job_run_completion=True)
```

Comment:
Perhaps we could add the number of retries with retry delay.
It could be passed into the constructor of the task. https://docs.prefect.io/api/latest/core/task.html#task-2
as:
```
dbt.DbtCloudRunJob(max_retries=3, retry_delay=datetime.timedelta(seconds=2))(cause=FLOW_NAME, job_id=DBT_CLOUD_BRAZE_JOB_ID, wait_for_job_run_completion=True)
```
^^ untested code

Reply:
Yeah, I added that to the prospecting flow but didn't add it here because this job was running every 10 minutes anyway. I could go either way for this flow.

Reply:
I don't feel strongly either. Fine to leave as-is. Good that G noticed this though.